recipe/meta.yaml

```diff
@@ -179,9 +179,10 @@ outputs:
         - cmake
         - m2-make   # [win]
         - posix     # [win]
+        # hack (around which bug?)
```

Comment:
This was noted as a hack in https://github.com/conda-forge/blas-feedstock/commit/a5df20db6d59e62a5bf94400e0cdadf4ea121396, can we document what's necessary to remove it?